templates/template.yaml

```diff
@@ -300,6 +311,14 @@ Resources:
         JobTableName: !GetAtt DDBStack.Outputs.JobTable
         JobTableStreamArn: !GetAtt DDBStack.Outputs.JobTableStreamArn
         StateMachineArn: !GetAtt StateMachineStack.Outputs.StateMachineArn
+  VpcStack:
+    Type: AWS::CloudFormation::Stack
+    Condition: DeployVpc
+    Properties:
+      TemplateURL: ./vpc.yaml
+      Parameters:
+        FlowLogsGroup: !Ref FlowLogsGroup
+        FlowLogsRoleArn: !Ref FlowLogsRoleArn
```

Comment:
Should we allow passing down the other 2 params too? (the IP blocks to be used). Probably explaining those defaults will be ignored if you specify subnet/sec groups for existing VPC.

Reply:
I wasn’t sure on that one. In my mind if you’re customising those, you may as well be using your own VPC?

Reply:
The problem is that maybe you want the VPC but those CIDRs are already used? I guess an escape would be to deploy the VPC separately if that would be the case, but I think we also want to remove the links to that 😢

Reply:
CIDR blocks already being used is only a problem if you can't have VPCs with overlapping IPv4 CIDR block (e.g. you intend to peer the S3F2 VPC?) but at that point you should be using your own VPC rather than the solution provided one because you'll be amending a CFN managed resource. so I still don't see a use case where you need to be able to customise all the params but still use the solution provided VPC rather than your own existing one?